src/Input/Input.js

```diff
@@ -89,6 +129,22 @@ module.exports = kind(
 	},
 
 	/**
+	* @private
+	*/
+	components: [
+		{kind: TooltipDecorator, autoShow: false, components: [
+			{name: 'tooltip', kind: Tooltip, floating: false, position: 'below'}	// To Do : It's position has dependency with ENYO-2709.
```

Comment:
Taking a deeper look at this now - I thought we wanted the tooltip to display on the side of the input? Have you tried experimenting with the layout some more? It does not appear we are using `TooltipDecorator` correctly here, but since we are controlling the showing/hiding of the tooltip directly, we may not actually need to use it.